tests/upload_test.go

```diff
@@ -165,3 +166,129 @@ func uploadImage(portForwardURL, token string, expectedStatus int) error {
 
 	return nil
 }
+
+var _ = Describe("Block PV upload Test", func() {
+	var (
+		pvc *v1.PersistentVolumeClaim
+		err error
+
+		uploadProxyURL string
+		portForwardCmd *exec.Cmd
+
+		pv           *v1.PersistentVolume
+		storageClass *k8sv1.StorageClass
+	)
+
+	f := framework.NewFrameworkOrDie(namespacePrefix)
+
+	BeforeEach(func() {
+		pod, err := utils.FindPodByPrefix(f.K8sClient, "cdi", "cdi-block-device", "kubevirt.io=cdi-block-device")
+		Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("Unable to get pod %q", "cdi"+"/"+"cdi-block-device"))
+
+		nodeName := pod.Spec.NodeName
+
+		By(fmt.Sprintf("Creating storageClass for Block PV"))
+		_, err = f.CreateStorageClassFromDefinition(utils.NewStorageClassForBlockPVDefinition("manual"))
```

Comment:
you are not capturing the storage class in storageClass so it doesn't get cleaned up.

Reply:
:-(